source/adapters/hip/enqueue.cpp

```diff
@@ -85,6 +85,62 @@ void simpleGuessLocalWorkSize(size_t *ThreadsPerBlock,
     --ThreadsPerBlock[0];
   }
 }
+
+ur_result_t setHipMemAdvise(const void *DevPtr, size_t Size,
+                            ur_usm_advice_flags_t URAdviceFlags,
```

Comment:
can the `Size` be `const`?
Also, in  `urEnqueueUSMAdvise` the argument is `size_t size`, here is  `size_t Size`.